### ISSUE TITLE
Missing textfield a11y description feedback form

### ISSUE THREAD

**Issue by f1sh1918** [digitalfabrik/integreat-app#3066]
**Is your feature request related to a problem? Please describe.**
The ios a11y inspector reported that the textfields (comment and e-mail) in the feedbackform don't have a description

**Describe the solution you'd like**
- add a `accessibilityLabel` to both text forms
